Source/ACE/ACE.csproj

```diff
@@ -235,6 +236,9 @@
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
     <None Include="App.config" />
+    <None Include="Config.json">

```

Comment:
Config.json is being intentionally being left out of the csproj file and should not be included here